src/plugins/conceptPlugin.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2017-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import ReactDOMServerStream from 'react-dom/server';
+import { Glossary } from 'ndla-ui';
+import { fetchConcept } from '../api/conceptApi';
+
+export default function createConceptPlugin() {
+  const fetchResource = (embed, headers) => fetchConcept(embed, headers);
+
+  const embedToHTML = embed => {
+    const {
+      id,
+      title: { title },
+      content: { content },
+      copyright,
+    } = embed.concept;
+
+    const messages = {
+      ariaLabel: 'Vis begrep beskrivelse',
```

Comment:
Trenger i18n her. Se her for hvordan det kan gjøres: https://github.com/NDLANO/article-converter/blob/master/src/plugins/imagePlugin.js#L16